src/main/java/org/gridsuite/filter/server/utils/FilterType.java

```diff
@@ -12,6 +12,7 @@
  */
 public enum FilterType {
     SCRIPT(""),
+    FILTER(""),
```

Comment:
so, FILTER here is a summary for LINE, GENERATOR, LOAD ... values
Maybe, we can have two enums ? one with just SCRIPT and FILTER, and another used in case of first enum is FILTER ?

Reply:
Agree, we are going to merge to PR as it is and do this refactoring in another one.